### PR TITLE
Refactored cli to return errors

### LINKED ISSUE #23: using deprecated call from codegangsta cli app
> DEPRECATED Action signature.  Must be \`cli.ActionFunc\`.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/codegangsta/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature

### PR DESCRIPTION
**Pull request by packrat386**
The old signature used in this application has been deprecated, see:
https://github.com/codegangsta/cli/blob/master/CHANGELOG.md#deprecated

closes #23

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -131,19 +131,21 @@ func main() {
 					Usage: "generate a migration that will not be wrapped in a transaction when run",
 				},
 			},
-			Action: func(c *cli.Context) {
+			Action: func(c *cli.Context) error {
 				if len(c.Args()) == 0 {
-					println("migration name not given! try `pgmgr migration NameGoesHere`")
-				} else {
-					displayErrorOrMessage(pgmgr.CreateMigration(config, c.Args()[0], c.Bool("no-txn")))
+					fmt.Println("migration name not given! try `pgmgr migration NameGoesHere`")
```
Comment (rnubel) [inline main.go]: This probably ought to be returned as a cli.NewExitError too, since it's an error case.

**Comment by rnubel** [comment]
Thanks for this! Looks great. I just left one comment about an error path that the tool wasn't properly returning an error code for.

**Comment by packrat386** [comment]
Fixed that. I wasn't sure it that was supposed to be an error condition or not so I left is as close to what it was as possible. I just tend to remove usage of the builtin `println` since it's use is discouraged.

**Comment by rnubel** [comment]
Roger that.

**Pull request merged**